components/nimbus/android/src/main/java/org/mozilla/experiments/nimbus/Nimbus.kt

```diff
@@ -540,24 +540,24 @@ open class Nimbus(
         enrollmentChangeEvents.forEach { event ->
             when (event.change) {
                 EnrollmentChangeEventType.ENROLLMENT -> {
-                    NimbusEvents.enrollment.record(mapOf(
-                        NimbusEvents.enrollmentKeys.experiment to event.experimentSlug,
-                        NimbusEvents.enrollmentKeys.branch to event.branchSlug,
-                        NimbusEvents.enrollmentKeys.enrollmentId to event.enrollmentId
+                    NimbusEvents.enrollment.record(NimbusEvents.EnrollmentExtra(
```

Comment:
The previous Glean recording API is deprecated, and there's a new more strongly-typed one documented [here](https://mozilla.github.io/glean/book/reference/metrics/event.html#recording-api).